django/contrib/admin/options.py

```diff
@@ -829,6 +829,15 @@ def get_action(self, action):
             description = capfirst(action.replace('_', ' '))
         return func, action, description
 
+    def get_empty_value_display(self):
```

Comment:
It would work like `allow_tags` and `admin_order_field` which are attributes of the fields added to the model.